assets/js/models/Simulator.coffee

```diff
@@ -0,0 +1,14 @@
+define (require) -> 
+  _BaseModels = require '_BaseModels'
+
+  class Simulator extends _BaseModels.ParentModel 
+    validate: (attrs, opts) =>
+      if opts.isEmpty
+        if !attrs.name? || !attrs.type_number? || !attrs.price?
+          'Please fill every field!'
+        else if !(Number.isInteger parseInt attrs.price)
+          ' Please fill the price field just number!'

```

Comment:
Nem jó, ha bepasszolod az isEmpty-t, akkor csak azt nézi, hogy nem null/undefined, ennek nem így kell működnie! Ha van isEmpty, akkor is ugyanúgy nézi, hogy validak-e a mezők PLUSZ azt is, hogy léteznek-e (mert alapból, ha nem léteznek, vagy undefined/null-ok, akkor nem kell elbuknia a validate-nek)